www/class/config-generate/broker.class.php

```diff
@@ -233,6 +234,18 @@ private function generate($poller_id, $localhost)
                         $object[$key][$subvalue['config_group_id']][$res[0]][(int)$subvalue['fieldIndex']][$res[1]] =
                             $subvalue['config_value'];
                         $subValuesToCastInArray[$subvalue['config_group_id']][] = $res[0];
+
+                        if (strcmp(
+                                $object[$key][$subvalue['config_group_id']]['name'],
+                                "forward-to-anomaly-detection"
+                            ) == 0
+                            && strcmp(
+                                $object[$key][$subvalue['config_group_id']]['path'],
+                                "/usr/share/centreon-broker/lua/centreon-anomaly-detection.lua"
```

Comment:
you shouldn't hardcode the broker path, as users can choose something else (eg: source install)

Reply:
This is installed by the module. The user can’t change this value